controllers/k8ssandra/validate_cluster_update.go

```diff
@@ -0,0 +1,37 @@
+package k8ssandra
+
+import (
+	"fmt"
+
+	api "github.com/k8ssandra/k8ssandra-operator/apis/k8ssandra/v1alpha1"
+	"github.com/k8ssandra/k8ssandra-operator/pkg/utils"
+)
+
+func validateK8ssandraCluster(kc api.K8ssandraCluster) error {
+	oldDCs := make([]string, 0)
+	for dcName, _ := range kc.Status.Datacenters {
+		oldDCs = append(oldDCs, dcName)
+	}
+	newDcs := make([]string, 0)
```

Comment:
question: Why aren't these using the DcAdded and DcRemoved, when they were made exported methods?

Reply:
Because the logic is different.

Reply:
Well, then that means this isn't tested at all?